src/keyring/KeyringController.ts

```diff
@@ -18,6 +18,8 @@ import {
   MetaMaskKeyring as QRKeyring,
   IKeyringState as IQRKeyringState,
 } from '@keystonehq/metamask-airgapped-keyring';
+import Transport from '@ledgerhq/hw-transport';
```

Comment:
What is `Transport`? I'm not familiar with this concept

Reply:
Transport is the base type for all transports with ledger. I've only added it as @mcmire preferred the transport function parameter to be typed instead of having it as `any`